### ISSUE TITLE
Search function

### ISSUE THREAD

**Issue by jony0008** [PaulWoitaschek/MaterialAudiobookPlayer#571]
Would be great if Material Player had a search function so it would be easy to find the album/audiobook/song/chapter/author/bookmark without having to go through a big library manually.

**Comment by sandybeachit** [comment]
jony008 how would you propose this search function would work? Would it be a search box and then you type in say "star" and it would look for all entries which contain the work star such in (album/audiobook/song/chapter/author/bookmark)

Have you had the need to do this and would anyone else find this useful?

I have no need of this.

**Comment by jony0008** [comment]
@sandybeachit I think it could be in the toolbar and would work as you say. I have a need for it but I can't talk for anyone else.

**Comment by sandybeachit** [comment]
Is your requirement just to be able to search with chapters instead of 
having to manually go through each book to find what your looking for?

Also what media would you be searching through as I only use material 
player for audio books?

**Comment by jony0008** [comment]
IMO a good start would be to able to search through everything, like file names, titles, authors in the library and would be more useful than searching for chapters

**Comment by jony0008** [comment]
@sandybeachit On second thought. A good start would be a way to search book titles and authors in the library and also chapters if possible. File names I can search by using a file browser.

**Comment by Lionfox2** [comment]
I would like to add, that I'm also missing a search function. It would be enough If the function would look for entries in the title which contains the search term. I have approx. 40 audiobooks and it is always a bit annoying to find an audiobook.

**Comment by JRSPE** [comment]
Ditto the need for a search function.  Voice is a wonderful audiobook reader because of its simplicity/minimalism and currently works well for those who have a small library.  But it does need to add some search capability for those with larger libraries [80-90 books in my library].  In keeping with the minimalist scheme I think it should be able to search folder names, book titles, and authors, plus ? and * wildcards - nothing more - KISS.  Don't need complex search strings, content searches, collections, huge db indexes, etc.

**Comment by gbilling008** [comment]
Yes, please add either a search or sort by author. At present I have approx 470 audio books and I'm trying to recover my laptop's harddrive to add another 200 audiobooks. Sorting by author would make it so much easier to keep track of books in a series. Before, during and after listening. Oh, and thanks for a great app.
